packages/contentful/migrations/events/20230404114001-create-events-content-model.js

```diff
@@ -0,0 +1,804 @@
+      helpText:
+        "This box is automatically ticked if no output is added after 14 days from the event's end date.",
+      trueLabel: 'Yes',
+      falseLabel: 'No',
+    },
+  );
+
+  events.changeFieldControl('videoRecording', 'builtin', 'richTextEditor', {
+    helpText:
+      'If permanently unavailable box is ticked, any content you put here will be ignored.',
+  });
+
+  events.changeFieldControl(
+    'videoRecordingUpdatedAt',
+    'app',
+    'mqvX9KU5AthRTlnIRhNNh',
+    {
+      observedField: 'videoRecording',
+    },
+  );
+
+  events.changeFieldControl(
+    'presentationPermanentlyUnavailable',
+    'builtin',
+    'boolean',
+    {},
+  );
+
+  events.changeFieldControl('presentation', 'builtin', 'richTextEditor', {
+    helpText:
+      'If permanently unavailable box is ticked, any content you put here will be ignored.',
+  });
+
+  events.changeFieldControl(
+    'presentationUpdatedAt',
+    'app',
+    'mqvX9KU5AthRTlnIRhNNh',
+    {
+      observedField: 'presentation',
+    },
+  );
+
+  events.changeFieldControl(
+    'meetingMaterials',
+    'app',
+    '3TfrYX8zVOgFtKxro89UVA',
+    {
+      key1: 'title',
+      key2: 'url',
+      helpText:
+        'If permanently unavailable box is ticked, any content you put here will be ignored.',
+    },
+  );
+
+  events.changeFieldControl(
+    'meetingMaterialsPermanentlyUnavailable',
+    'builtin',
+    'boolean',
+    {
+      helpText:
+        "This box is automatically ticked if no output is added after 14 days from the event's end date.",
+      trueLabel: 'Yes',
+      falseLabel: 'No',
+    },
+  );
+
+  events.changeFieldControl('description', 'app', '2finDNk15g5UtOq4DaLNxv', {});
+  events.changeFieldControl(
+    'startDateTimeZone',
+    'app',
+    '2finDNk15g5UtOq4DaLNxv',
+    {},
+  );
+  events.changeFieldControl('endDate', 'app', '2finDNk15g5UtOq4DaLNxv', {});
+  events.changeFieldControl(
+    'endDateTimeZone',
+    'app',
+    '2finDNk15g5UtOq4DaLNxv',
+    {},
+  );
+  events.changeFieldControl('eventLink', 'app', '2finDNk15g5UtOq4DaLNxv', {});
+  events.changeFieldControl('googleId', 'app', '2finDNk15g5UtOq4DaLNxv', {});
+
+  const eventSpeakers = migration
+    .createContentType('eventSpeakers')
+    .name('Event Speakers')
+    .description('')
+    .displayField('title');
+  eventSpeakers
+    .createField('title')
+    .name('Title')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  eventSpeakers
+    .createField('team')
+    .name('Team')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkContentType: ['teams'],
+      },
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Entry');
+
+  eventSpeakers
+    .createField('user')
+    .name('User')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkContentType: ['externalAuthors'],
```

Comment:
reminder to myself: rebase to https://github.com/yldio/asap-hub/pull/2862 and add `users` here